index.js

```diff
@@ -10,56 +9,37 @@ client.on("ready", () => {
   console.log("Bot funcionando 🤖");
 });
 
-client.on("messageCreate", (msg) => {
-  if (msg.content.startsWith(prefixRedaction)) {
-    let args = msg.content.split(" ");
-    let argsSliced = args.slice(1).join(" ");
-    if (argsSliced.length === 0) {
-      msg.reply({
-        content: `Ops! Parece que você esqueceu de inserir argumentos 😬`,
-      });
-      if (argsSliced.length === 1) {
-        msg.reply({
-          content: `Ops! Parece que você esqueceu de inserir o nome do artigo 😬`,
-        });
-      }
-    } else {
-      msg.reply({
-        content: `${msg.author} acabou  de mandar o artigo "**${argsSliced}**" para a fila de revisão 🚀 \n\n Divirta-se, <@257316997707071491> 😁`,
-      });
-    }
+client.on("messageCreate", (message) => {
+  // Ignores bot messages
+  if (message.author.bot) return;
+
+  // Ignores commandName and extract arguments
+  const args = message.content.split(' ').slice(1);
+
+  if (args.length === 0) {
+    return message.reply({ content: `Ops! Parece que você esqueceu de inserir argumentos 😬` });
+  }
+
+  if (message.content.startsWith(prefixRedaction)) {
+    const postName = args.join(' '); // commandName postName
+
+    return message.reply({
+      content: `${message.author} acabou  de mandar o artigo "**${postName}**" para a fila de revisão 🚀 \n\n Divirta-se, <@257316997707071491> 😁`,
+    });
   }
 
-  if (msg.content.startsWith(prefixEditor)) {
-    let args = msg.content.split(" ");
-    let argsSliced = args.slice(2).join(" ");
+  if (message.content.startsWith(prefixEditor)) {
+    const postName = args.slice(1).join(' '); // commandName postStatus postName
+
+    const messages = {
+      '1': `Revisão do artigo "**${postName}**" liberada ✅`,
+      '2': `Revisão do artigo "**${postName}**" liberada com observações 👀`,
+      '1t': `Revisão do artigo "**${postName}**" liberada ✅ \n\nAguardando thumb 🖼`,
+      '2t': `Revisão do artigo "**${postName}**" liberada com observações 👀 \n\nAguardando thumb 🖼`,
+      '_default': 'Ops! Parece que você esqueceu de inserir argumentos 😬'
+    };
 
-    switch (args[1]) {
-      case "1":
-        msg.reply({
-          content: `Revisão do artigo "**${argsSliced}**" liberada ✅`,
-        });
-        break;
-      case "2":
-        msg.reply({
-          content: `Revisão do artigo "**${argsSliced}**" liberada com observações 👀`,
-        });
-        break;
-      case "1t":
-        msg.reply({
-          content: `Revisão do artigo "**${argsSliced}**" liberada ✅ \n\nAguardando thumb 🖼`,
-        });
-        break;
-      case "2t":
-        msg.reply({
-          content: `Revisão do artigo "**${argsSliced}**" liberada com observações 👀 \n\nAguardando thumb 🖼`,
-        });
-        break;
-      default:
-        msg.reply({
-          content: "Ops! Parece que você esqueceu de inserir argumentos 😬",
-        });
-    }
+    return message.reply({ content: messages[args[0]] || messages['_default'] });
```

Comment:
I think there's a better way to make this.

First of all I would validate the presence of the arguments before replying.
Then change the `_default` message to something clearer. Something like:
`Ops! O argumento que você informou é inválido. (Any emote here)`

We are using just one arg here, so I would change line `32` to:
```js
const postName = args.slice(1).join(' ');
const commandAction = args[0];
```

I would remove the comments too. I think the code is sufficiently clear, so it does not need it.

Reply:
Added presence of the arguments and changed `_default` message. [commit#b8727cf](https://github.com/Lavrudin/diolinux-team-bot/pull/1/commits/b8727cf73001b8c0096724ff1b5ca83b4adbbc6c)